MedicalDocManagment.WebUI/Controllers/api/AdminController.cs

```diff
@@ -104,6 +104,25 @@ private IHttpActionResult GetErrorResult(IdentityResult result)
             var result = await UsersManager.UpdateAsync(userInDb);
             return Ok(result);
         }
-        
+        [HttpDelete]
+        public async Task<HttpResponseMessage> DeleteUser(int id)
+        {
+            var user = await UsersManager.FindByIdAsync(id.ToString());
+            if (user != null)
+            {
+                if (!user.IsActive)
+                {
+                    return Request.CreateErrorResponse(HttpStatusCode.BadRequest, new HttpError("User already deleted."));
+                }
+                user.IsActive = false;
+                var result = await UsersManager.UpdateAsync(user);
+                if (result.Succeeded)
```

Comment:
When `result` is going to be `false` the method will be return `Not Found`. It isn't critical bug, but anyway. You may use Alexander's method ( `GetErrorResponse` )